projects/canopy/src/lib/forms/date/date-field.component.spec.ts

```diff
@@ -229,9 +229,9 @@ describe('LgDateFieldComponent', () => {
   it('sets unique identifiers for each input field', () => {
     fixture.detectChanges();
 
-    expect(/lg-input-year-\d{1,3}/.test(yearInput.nativeElement.id)).toBe(true);
```

Comment:
I wonder if the ids were specific for a reason 🤔

Reply:
I think it was just because the `LgInputComponent` would set the `id` by default using the internal `nextUniqueId` and when using a custom label outside of that component, there is no way to know what it is to set the `for` attribute correctly

Reply:
I can probably just remove that test now actually

Reply:
I did the same too 😄